### PR TITLE
Tiebreaker

### PR DESCRIPTION
**Pull request by kevintpeng**

### CODE REVIEW

File: js/paxos.js [hunk 1]
```diff
@@ -92,27 +92,51 @@ class Paxos {
...
+    var times = this.current_proposal.response_tuples.map(function(o){ return o.seq.time; });
+    var highest_sequence = Paxos.maxSequence;
+    var peers = this.comms.peers;
+    for (var peerIdx in peers) {
+      var peer = peers[peerIdx];
+      this.comms.sendDataToPeer(peer, 'COMMIT', highest_sequence);
+    }
+  }
+
+  static acceptCommit(proposal, current) {
```
Comment (jules2689) [inline js/paxos.js:121]: `shouldAcceptCommit` makes it clear its a check, I thought this was accepting it

File: js/paxos.js [hunk 2]
```diff
@@ -92,27 +92,51 @@ class Paxos {
...
+           Paxos.sequenceGt(proposal.seq, current.accepted_seq)
+  }
+
+  static sequenceGt(seq1, seq2) {
+    return (seq1.time > current.seq.time) ||
+           (seq1.time === current.seq.time && seq1.id > current.seq.id)
+  }
+
+  static maxSequence(sequences) {
+    max_seq = {time: 0, id: ''}
```
Comment (jules2689) [inline js/paxos.js:132]: doesn't this need to be `let max_seq = ...`

**Comment by nwoodthorpe** [comment]
Any chance of this getting merged? I'm blocked on this.

**Comment by jules2689** [comment]
I saw the notification for this and thought "Oh no, what did nat comment?"

**Pull request closed without merging**